src/main/serialization.ts

```diff
@@ -0,0 +1,39 @@
+import to_json from "fast-json-stable-stringify"
```

Comment:
This file was copied from the master branch. It replaces the private serialization methods on the `LibsodiumEncryptor` class.